src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1750,6 +1771,20 @@ def load_equipment_description(
     )
 
 
+def load_annulus_equipment_description(
+    document: DescriptionDocument,
+) -> case_description.AnnulusEquipmentDescription:
+    alfacase_to_case_description = {
+        "gas_lift_valves": load_gas_lift_valve_equipment_description,
+        "leaks": load_leak_equipment_description,
+    }
+    return _generate_description(
```

Comment:
Did you try to use auto loaders?

Maybe using directly `load_instance`.
I would expect all case descriptions that only use those `attrib_*` or are plain values to be loadeble by auto loaders.
I will like to see this file with as lest code possible like `case_to_alfacase`.